### DISCUSSION TITLE
Youtube live stream features not working!

### DISCUSSION THREAD

**Discussion by Nasimaktar99** [anasty17/mirror-leech-telegram-bot#921] [Ideas]
When try to use /ytdl command for youtube live stream video after choosing quality it shows :-  your download has been stopped due to: ERROR: ffmpeg exited with code -11

Log:-bot.helper.mirror_utils.download_utils.yt_dlp_download_helper - INFO - Downloading with YT-DLP: https://youtu.be/TBFPiAQab5g
2022-08-16 16:54:48,455 - bot.helper.mirror_utils.download_utils.yt_dlp_download_helper - ERROR - 

2022-08-16 16:54:48,456 - bot.helper.mirror_utils.download_utils.yt_dlp_download_helper - ERROR - ERROR: ffmpeg exited with code -11
2022-08-16 16:54:48,456 - bot - INFO - Cleaning Download: /usr/src/app/downloads/15924

**Comment by anasty17** [comment]
Ok i will fix this

**Reply by anasty17** [reply]
Try to restart the bot. It will work. But i will solve this issue soon but u must deploy again after i fix. I will post comment here after fix

**Comment by anasty17** [comment]
It should be fine now after deploying

**Reply by Nasimaktar99** [reply]
> It should be fine now after deploying

I've set Upstream Repo is your Repo & branch h-code, then also need to refork & redeploy or restarting bot will works?
